### ISSUE TITLE
create word list - "separated by enter key"

### ISSUE THREAD

**Issue by dakom** [ji-devs/ji-cloud-flowcharts#1]
This is no longer the case, rather, we have individual cells that are navigated like regular html forms (e.g. tab key switches between them)

**Comment by corinnewo** [comment]
Yes, it is no longer the one field. A nice-to-have would be to copy and paste from a list, and the list would be broken at the end of each line, and assigned to a cell. But, for the MVP, we can do without this.
